playbooks/roles/notify-failure-rocket/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Clone ardana-bm-info repository
+  git:
+    repo: "{{ rc_auth_info_git_url }}"
+    dest: "{{ rc_auth_info_dir }}"
+    version: "master"
+    force: yes
+
+- name: Load rocketchat auth variables
+  include_vars: "{{ rc_auth_info_dir }}/rc.yml"
```

Comment:
can't we just ``set_fact`` ``rc_token`` by fetching it from a url?
``https://gitlab.suse.de/cloud-qe/ardana-bm-info/raw/master/rc.yml`` seems wide open to me behind the vpn.

Reply:
We could I guess.

Reply:
or simply use the lookup in the defaults.

Assuming you then make the arguments of the rocket chat call part of the default, then it becomes simple to reuse this across the whole org.